cgroups/manager/manager.go

```diff
@@ -0,0 +1,19 @@
+package manager
+
+import (
+	"github.com/docker/libcontainer/cgroups"
+	"github.com/docker/libcontainer/cgroups/fs"
+	"github.com/docker/libcontainer/cgroups/systemd"
+)
+
+func NewCgroupManager(cgroups *cgroups.Cgroup) cgroups.Manager {
```

Comment:
Not really related to this PR, but I'm not a huge fan of how this object is not really initialized until Apply() is called. Maybe add a TODO to make this to the equivalent of Apply() at some point? Feel free to add the TODO for me.